CHANGELOG.md

```diff
@@ -67,7 +67,30 @@
   details.
 </details>
 
-## v1.32.0 - TBD
+## v1.33.0 - TBD
+
+## v1.32.0 - 2021-10
+
+### [BigQuery](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/README.md)
+
+* feat(generator): generate a SetIamPolicy() overload with an OCC loop ([#7276](https://github.com/googleapis/google-cloud-cpp/pull/7276))
+
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+* docs(bigtable): add documentation for no channel refreshes ([#7373](https://github.com/googleapis/google-cloud-cpp/pull/7373))
+
+### [IAM](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/iam/README.md)
+
+* feat(generator): generate a SetIamPolicy() overload with an OCC loop ([#7276](https://github.com/googleapis/google-cloud-cpp/pull/7276))
+* doc(iam): use SetIamPolicy() read-modify-write loop in sample ([#7288](https://github.com/googleapis/google-cloud-cpp/pull/7288))
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat(pubsub): use `google::cloud::Options` to configure the library.
+  `pubsub::PublisherOptions`, `pubsub::SubscriberOptions`, and related
+  functions are deprecated. We have not set a date to remove them.
```

Comment:
s/remove them./remove them, if we ever do, we plan to give you a year notice./

Or a better way to say that if/when we set a date, it will be at least a year after we make the decision to remove them.

Reply:
Done.